docs/loot/recipes.md

```diff
@@ -0,0 +1,625 @@
+---
+title: Recipes
+mention:
+	- Ciosciaa
+---
+
+*Last updated for Version 1.17.41*
+
+Recipes are the means of handling several item transactions, namely those occurring in crafting tables, furnaces and campfires, and brewing stands.
+
+<CodeHeader>BP/recipes/crafting/weapons/cold_steel_sword.json</CodeHeader>
+```json
+{
+	"format_version": "1.17.41",
+	
+	"minecraft:recipe_shaped": {
+		"description": {
+			"identifier": "lichbane:cold_steel_sword"
+		},
+		"tags": ["crafting_table", "altar"],
+
+		"pattern": [
+			"X",
+			"X",
+			"I"
+		],
+		"key": {
+			"X": "lichbane:cold_steel",
+			"I": "minecraft:stick"
+		},
+		"result": "lichbane:cold_steel_sword"
+	}
+}
+```
+
+::: tip NOTE
+Anvil interactions are handled within an [item definition](https://wiki.bedrock.dev/items/items-16.html). Smithing table and loom transactions are currently unavailable.
+:::
+
+All recipes are stored in the `recipes` folder in the behavior pack root. The files can be named and organized under any folder hierarchy as desired. No experimental toggles are required to use recipes or any of their features.
+
+## Shared Properties and Structures
+### Format Version
+The format version is intended to version the schema used for the body of a recipe. It is provided with the top-level `"format_version"` property.
+
+<CodeHeader>#/</CodeHeader>
```

Comment:
This is weird. Now its just #/

Reply:
I don't follow this one. This is structurally consistent for any code snippet.